blog/_posts/2024-06-25-history-ele-raid-buffs.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: The Historical Precedent for an Elemental Raid Buff
+#            dd/mm/yyyy
+last_update: 25/06/2024
+excerpt: Elemental Shaman's place as a raid buff provider.
+author: Lucenia
+toc: true
+big_article: true
+---
+
+> Lucenia is a 15-year veteran of Elemental Shaman currently Cutting-Edge raiding in [\<Noted\>](https://raider.io/guilds/us/akama/Noted) on US-Akama. She streams mythic raiding, mythic+, and PTR/beta testing content at https://twitch.tv/Lucenia.
```

Comment:
Any reason to not make your twitch url an actual link? I'd say: add it

Reply:
It was rendering as a link in the GH markdown editor but if it won't on SE&L then sure 😄